src/injection.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
not sure on this icon 🤣 it is a little... on the nose

Reply:
honestly.. ive been looking for another one too because I hate the use of needles.. but I cant think of a better one to use!! any ideas?!

Reply:
how about this one ? https://github.com/n3wscott/sockeye-react/pull/6/commits/f1ee4d9c221751c7a09655ea904737006be63974

Reply:
<img width="459" alt="Screen Shot 2021-09-16 at 11 12 09 AM" src="https://user-images.githubusercontent.com/60796676/133663597-a477f53a-9cb9-450c-b317-ecb678795eb8.png">